Library/Homebrew/cmd/commands.rb

```diff
@@ -8,51 +8,66 @@ module Homebrew
   module_function
 
   def commands
-    if ARGV.include? "--quiet"
-      cmds = internal_commands + external_commands
-      cmds += internal_developer_commands
-      cmds += HOMEBREW_INTERNAL_COMMAND_ALIASES.keys if ARGV.include? "--include-aliases"
-      puts Formatter.columns(cmds.sort)
-    else
-      # Find commands in Homebrew/cmd
-      puts "Built-in commands"
-      puts Formatter.columns(internal_commands)
-
-      # Find commands in Homebrew/dev-cmd
-      puts
-      puts "Built-in developer commands"
-      puts Formatter.columns(internal_developer_commands)
-
-      # Find commands in the path
-      unless (exts = external_commands).empty?
+    command = CommandsCommand.new
+    command.commands
+  end
+
+  class CommandsCommand < CommandOptions
```

Comment:
Since this class implements the whole behaviour of `brew commands`, rather than just describing its options, it seems strange to say that a `commands` command is an instance of `CommandOptions` — maybe the superclass should just be renamed `Command`?